### ISSUE TITLE
Improve Form Usability and Accessibility

### ISSUE THREAD

**Issue by aditi-jadhav-arch** [Creators-Space/Creators-Space#358]
**Description**

Some forms in the project are missing key usability and accessibility elements:

Login and Signup Forms – Labels exist, but the input fields don’t have placeholder text. This makes the forms less intuitive.

"Get in Touch" Form (Footer) – Input fields are missing labels, which reduces accessibility and may confuse users.

**Why This Matters**

Placeholders provide inline guidance, improving usability.

Labels are essential for accessibility, especially for screen readers.

Together, these changes make forms more user-friendly, inclusive, and aligned with UX best practices.

**Screenshots of current forms:**

<img width="500" height="500" alt="Image" src="https://github.com/user-attachments/assets/49e7691a-02b6-465e-84ad-fbfb874d0008" />
<img width="500 height="574" alt="Image" src="https://github.com/user-attachments/assets/ccf65d99-98e2-458b-ad48-1fb151c02e6c" />
<img width="500" height="500" alt="Image" src="https://github.com/user-attachments/assets/41646edf-252a-4901-a840-2dcb5ff4ffe1" />
<img width="500" height="500" alt="Image" src="https://github.com/user-attachments/assets/1b64905d-a46f-4813-8cb7-ab7058585e5d" />

**Comment by aditi-jadhav-arch** [comment]
@vishanurag 
I would like to work on this issue. Please assign it to me.
